### DISCUSSION TITLE
Get ready for 1.1

### DISCUSSION THREAD

**Discussion by not-nef** [sumeshir26/TimerX#85] [General]
Im gonna add the ui for the hidden settings tomorrow and then we can release!

**Comment by lg-nineoneninethreenine** [comment]
Also, let's try adding an Easter egg in the logo that shows the people who contributed to this repo. You could see Windows 3.1 Easter eggs to see how it could work out, but don't add animation anywhere. @sumeshir26 can we make a Slack community? Just asking, since that would seem more inviting for people.

**Comment by not-nef** [comment]
> Also, let's try adding an Easter egg in the logo that shows the people who contributed to this repo. You could see Windows 3.1 Easter eggs to see how it could work out, but don't add animation anywhere. @sumeshir26 can we make a Slack community? Just asking, since that would seem more inviting for people.

Can you share like a picture of what you are thinking of?

**Reply by lg-nineoneninethreenine** [reply]
https://youtu.be/8R27RMBIk20

Like that, except don't add the animation, add it if you can and if it doesn't slow the app down. Make sure it's kinda like clicking the app logo in a hidden point. @sumeshir26 I recommend that we make a Slack community since it's easy for us to create a secret chat. The team chat seems too open, that's why I asked for a community.

**Comment by sumeshir26** [comment]
@not-nef Can you also fix the blur bug before release?

**Reply by not-nef** [reply]
The linux thing? I dont know why it doesnt work for linux so probably no. I will add a toggle for it tho.

**Reply by lg-nineoneninethreenine** [reply]
That's what we need.
